helm/oncall/values.yaml

```diff
@@ -708,3 +708,12 @@ ui:
     tag: dev
   # Additional env vars for the ui container
   env: {}
+
+prometheus:
+  enabled: false
+  extraScrapeConfigs: |
+    - job_name: 'oncall-exporter'
+      metrics_path: /metrics/
+      static_configs:
+        - targets:
+          - oncall-dev-engine.default.svc.cluster.local:8080
```

Comment:
I set it this way so that on by default github actions + local env you just need to set `enabled: true` for prometheus then it starts to scrape from `oncall-dev-engine` correctly. Alternatively we could skip `extraScrapeConfig` here and move it to `.github/helm-values.yml`. Or set some other URL that will work for prod helm chart instead and then override in `.github/helm-values.yml`? 

Could you help me with that? :)

Reply:
We talked with @joeyorlando and we agreed to go with https://github.com/grafana/oncall/pull/3583/commits/fd02e6a76170529f0576b0b07ebb994f056b370d